demisto_sdk/commands/lint/linter.py

```diff
@@ -1601,15 +1612,28 @@ def _get_docker_images_for_lint(
 
             self._check_native_image_flag(docker_image_flag)
 
-            if native_image := self._get_native_image_name_from_config_file(
-                docker_image_flag
-            ):
+            supported_section = docker_image_flag
+            if docker_image_flag == DockerImageFlagOption.NATIVE_TARGET.value:
+                supported_section = DockerImageFlagOption.NATIVE_DEV.value
+
+            native_image = self._get_native_image_name_from_config_file(
+                supported_section
+            )
+            if native_image:
```

Comment:
you can use the walrus operator as I did before: 
```suggestion
            if native_image := self._get_native_image_name_from_config_file(
                supported_section
            )
```